utils/python/CIME/utils.py

```diff
@@ -21,8 +21,9 @@ def expect(condition, error_msg, exc_type=SystemExit):
     """
     if (not condition):
         # Uncomment these to bring up a debugger when an expect fails
-        #import pdb
-        #pdb.set_trace()
+        if logger.isEnabledFor(logging.DEBUG):
```

Comment:
I got tired of commenting or uncommenting these lines.  Now if you run with --debug they are enabled but not otherwise.